CMakeLists.txt

```diff
@@ -133,6 +133,18 @@ if(NANO_STACKTRACE_BACKTRACE)
   endif()
 endif()
 
+option(NANO_TRACING "Enable trace logging" OFF)
+# Check if the build type is Debug, and if so, enable NANO_TRACING by default
+if(CMAKE_BUILD_TYPE STREQUAL "Debug")
```

Comment:
This forces NANO_TRACING to be always on when in Debug mode. But it would be more flexible to just set the default value to ON in Debug mode rather than force the final value. Like this, I think:

```
if(CMAKE_BUILD_TYPE STREQUAL "Debug")
option(NANO_TRACING "Enable trace logging" ON)
else()
option(NANO_TRACING "Enable trace logging" OFF)
endif()
```

Reply:
Yeah, this looks like a better way of doing it.